phone.py

```diff
@@ -37,8 +42,12 @@ def get_string(self):
         return phone_digit_shape
 
     def get_number(self, symbol):
-        """
-          Should convert letter to digit (see image in the README.md)
-          clue 1: use a dictionary.
-        """
-        pass
\ No newline at end of file
+        phone_keyboard_mapper ={'a':1, 'b':1,'c':1,'d':2,'e':2,'f':2,'g':3,'h':3,'i':3,'j':4,'k':4,'l':4,'m':5,'n':5,'o':5,'p':6,'q':6,'r':6,'s':7,'t':7,'u':7,'v':8,'w':8,'x':8,'y':9,'z':9, 
+
+        }
+        return (phone_keyboard_mapper.get(symbol.lower(),symbol))
+
+        #"""
+          #Should convert letter to digit (see image in the README.md)
+          #clue 1: use a dictionary.
+        #"""
```

Comment:
These lines should be the first one in the method `get_number`

It is describing what the `get_number` method is doing.

Reply:
Aquí